styles.css

```diff
@@ -0,0 +1,140 @@
+@import url("https://rsms.me/inter/inter.css");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Inter", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+}
+
+/* navBar */
+
+ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  font-weight: 500;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+li {
+  padding: 1%;
+  color: #172b4d;
+}
+
+a {
+  text-decoration: none;
+  color: #172b4d;
+}
+
+/* the whole content */
+
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  background-image: url(./utils/Header-llustration-desktop@2x.png);
```

Comment:
Kindly ensure that, following the design, the background image shows up properly. Not only does it look better, but you'll also get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible. So, sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit.

Follow the original vision if it's pretty and works, it will be a wonderful website.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185387739-da0f66a3-badb-4ab5-8a90-166c268eaa0d.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/185387760-54618642-7cd6-4546-a847-129fc11945b7.png)